python/ray/train/torch/torch_predictor.py

```diff
@@ -84,7 +84,8 @@ def from_checkpoint(
                 ``TorchTrainer`` run.
             model: If the checkpoint contains a model state dict, and not
                 the model itself, then the state dict will be loaded to this
-                ``model``.
+                ``model``. If the checkpoint already contains the model itself,
+                this model argument will be discarded.
```

Comment:
We might want to give a warning in this case if the model is already in the checkpoint and this parameter is specified -- I feel like the user will want to know if their `model` argument is ignored :)

Reply:
that's a good idea. Added it.